src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/DistributedCacheTagHelperFormatter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
+{
+    /// <summary>
+    /// Implements <see cref="IDistributedCacheTagHelperFormatter"/> by serializing the content
+    /// in UTF8.
+    /// </summary>
+    public class DistributedCacheTagHelperFormatter : IDistributedCacheTagHelperFormatter
```

Comment:
The goal of the extra service is to provide an extensibility point in the html serialization only (gzip, encryption, ...)